src/playground/addons/story-utils/index.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-param-reassign */
 import iconPathEc from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 import iconPathEu from '@ecl/resources-eu-icons/dist/sprites/icons.svg';
-import iconSocialPath from '@ecl/resources-ec-social-icons/dist/sprites/icons-social.svg';
+import iconMediaSocialPath from '@ecl/resources-social-media-icons/dist/sprites/icons-social-media.svg';
```

Comment:
icons from `@ecl/resources-ec-social-icons` are still used in some components (namely social media follow and social media share).
So we should have both resources available: footer uses icons from `@ecl/resources-social-media-icons`, the other two from `@ecl/resources-ec-social-icons`

Reply:
I added all 3 cases in this function (based on social-media, social and fallback cases).